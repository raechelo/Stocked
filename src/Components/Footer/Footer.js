import React from 'react'
import cleaver from '../../assets/cleaver.svg';

export default function Footer() {
  return (
    <footer className="Footer">
      <h5>difficulty:
        <img alt="cleaver" src={cleaver} />
        <img alt="cleaver" src={cleaver} />
        <img alt="cleaver" src={cleaver} />
      </h5>
    </footer>
  )
}
