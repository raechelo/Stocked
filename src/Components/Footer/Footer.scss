.Footer {
  position: absolute;
  bottom: -1rem;
  left: 45%;

  h5 {
    img {
      width: 2rem;
    }
  }
}