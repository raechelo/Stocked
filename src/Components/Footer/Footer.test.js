import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});