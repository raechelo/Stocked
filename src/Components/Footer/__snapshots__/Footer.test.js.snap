// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot with all the data passed in 1`] = `ShallowWrapper {}`;
